modules/programs/neovim.nix

```
@@ -226,22 +226,26 @@ in {
         settings = mkOption {
           type = jsonFormat.type;
           default = { };
-          example = literalExample ''
-            {
-              "suggest.noselect": true,
-              "suggest.enablePreview": true,
-              "suggest.enablePreselect":false,
-              "suggest.disableKind": true,
-              "languageserver": {
-                "haskell": {
-                  "command": "haskell-language-server-wrapper",
-                  "args": ["--lsp"],
-                  "rootPatterns": ["*.cabal", "stack.yaml", "cabal.project", "package.yaml", "hie.yaml"],
-                  "filetypes": ["haskell", "lhaskell"]
-                }
-              }
-            }
-          '';
+          example = {
```

```suggestion
          example = literalExample ''
            {
```

Sorry, last thing! You need to keep the literalExample when using attribute sets or lists in examples.

---

Done